authui/src/authflowv2/customSelect.ts

```diff
@@ -344,7 +344,7 @@ export class CustomSelectController extends Controller {
   renderTrigger() {
     const option =
       this.optionsValue.find((option) => option.value === this.value) ??
-      this.optionsValue.at(0);
+      this.optionsValue[0];
```

Comment:
ci failed because typescript thinks `this.optionsValue[0]` is always not `undefined`.
It seems you need to explicitly write the type of `option` as `SearchSelectOption | undefined`.